guide/README.md

```diff
@@ -6,7 +6,7 @@ collection of features relating to pitch and rhythm.
 
 The Turtle Blocks guide is a good place to start learning about the
 basics. In this guide, we illustrate the musical features by walking
-the reader through numerous examples.
+the reader through numerous examples.Recommended to read Guide to Programming with Turtle Art.(github.com/sugarlabs/turtleblocksjs/blob/master/guide/README.md)
 
 A short [Guide to
```

Comment:
Can you style it like this example below?

[text](Link)

Reply:
brackets, then parenthesis

Reply:
> brackets, then parenthesis

Hi @pikurasa , I have styled it like above. How can I contribute to this PR ( I can't see here to contribute) or should I create a New PR ?

Reply:
I suppose a new PR would be the easiest thing to do.

I have not heard back from the original contributor...

Reply:
Okk , done

Reply:
Sorry. I overlooked this one.

It is still a bit off:
(1) There should be a space after the period: "...numerous examples. Recommended..."
(2) The link syntax if wrong. [description](URL) is the proper syntax.
(3) Please flow the text so it is maximum of 80 characters per line.

Also, the wording itself is a bit awkward. Perhaps "We recommend that you read..."